src/index.js

```diff
@@ -1,11 +1,23 @@
 var bodyParser = require('body-parser');
 var app = require('express')();
 var handleWfhRequest = require('./handle-wfh-request');
+var parseText = require('./parse-text')
 
 app.use(bodyParser.urlencoded({ extended: true }));
 app.listen(process.env.PORT);
 
 app.post('/wfh', (req, res) => {
 	res.send();
-	handleWfhRequest(req.body.user_id, req.body.response_url);
+	if (parseText.checkIfClear(req.body.text))
```

Comment:
Currently, if a user types `/wfh tomorrow` twice - the server will delete the event for that day and send a similar message back to slack saying they are going to the office on tomorrow's date after the second command. The toggle works correctly regardless of date or time interval so I don't think the experiences are asymmetrical. You do not have to use `/wfh clear tomorrow` to remove tomorrow's wfh event. However, if you think that it is more intuitive to replace the toggle function with a function that adds a wfh event if one is not there but does not handle deletion, it may make sense to make this change.